DDDEastAnglia/Controllers/SponsorLogoService.cs

```diff
@@ -0,0 +1,21 @@
+using System.Web.Mvc;
+using DDDEastAnglia.DataAccess;
+
+namespace DDDEastAnglia.Controllers
+{
+    public class SponsorLogoService
+    {
+        private readonly ISponsorRepository sponsorRepository;
+
+        public SponsorLogoService(ISponsorRepository sponsorRepository)
+        {
+            this.sponsorRepository = sponsorRepository;
+        }
+
+        public FileContentResult Get(int sponsorId)
```

Comment:
A service shouldn't return a framework type, that should be for the Controller to decide.